packages/charts/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -652,7 +659,7 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
     let xBarScale: any;
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const yBarScale: any = d3ScaleLinear()
-      .domain([0, this._yMax])
+      .domain([this.props.supportNegativeYValues ? this._yMin : 0, this._yMax])
```

Comment:
>.domain([this.props.supportNegativeYValues ? this._yMin : 0, this._yMax]) [](http://example.com/codeflow?start=6&length=73)

Can you add screenshot for the following cases in description.

All y positive.
All y positive and yMin/yMax are set.
Some y positive, some y negative.
Some y positive, some y negative and yMin/yMax are set.
All y negative.
All y negative and yMin/yMax are set. 
yMax > 0 and yMax < 0

Reply:
Okay